treeherder/services/pulse/consumers.py

```diff
@@ -210,6 +220,11 @@ def on_message(self, body, message):
             store_pulse_tasks.apply_async(
                 args=[body, exchange, routing_key, self.root_url], queue='store_pulse_tasks'
             )
+            if 'task-completed' in exchange and '.proj-mozci.' in routing_key:
+                store_pulse_tasks_classification.apply_async(
```

Comment:
This part seems pretty important to test, but it may be hard to mock the context around.

It would be pretty interesting though, as you could directly validate the routes payload we are using (with `proj-mozci` inside).